buildSrc/src/main/java/org/elasticsearch/gradle/http/WaitForHttpResource.java

```diff
@@ -145,6 +145,7 @@ HttpURLConnection buildConnection(SSLContext ssl) throws IOException {
         configureSslContext(connection, ssl);
         configureBasicAuth(connection);
         connection.setRequestMethod("GET");
+        connection.setRequestProperty("Accept", "text/plain; charset=UTF-8");
```

Comment:
this prevents defaulting Accept header to `text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2` by `HttpUrlConnection`